### PR TITLE
Just some sanity checks to the blockade code

### PR DESCRIPTION
**Pull request by Hraelzyr**
Will change it in earnest as the main campaign for Proteron happens, for now just standing < 0 = hostile

### CODE REVIEW

File: dat/events/proteron/blockade.lua [hunk 1]
```diff
@@ -36,14 +36,15 @@ local function spawn_fleet( pos )
...
+   local stand, _str = faction.get("Proteron"):playerStanding()
    for k,s in ipairs(ships) do
       local leader = plts[1]
       local p = pilot.add( s, "Proteron", pos, nil, {ai="guard"} )
       local aimem = p:memory()
       aimem.enemyclose    = 10e3
       aimem.guarddodist   = 10e3
       aimem.guardreturndist = 15e3
-      p:setHostile(true) -- TODO change this when something can make them non-hostile
+      p:setHostile(stand<0)
```
Comment (bobbens) [inline dat/events/proteron/blockade.lua:47]: Strictly speaking, I don't think this is necessary, because this is the default behaviour.

Comment (Hraelzyr) [reply dat/events/proteron/blockade.lua:47]: Ah, I see, fair enough

File: dat/events/proteron/blockade.lua [hunk 2]
```diff
@@ -71,7 +72,8 @@ end
 
 function heartbeat( proteron_blockade )
    local pp = player.pilot()
-   for k,p in ipairs(proteron_blockade) do
+   local stand, _str = faction.get("Proteron"):playerStanding()
+   for k,p in ipairs(proteron_blockade) and stand < 0 do
```
Comment (bobbens) [inline dat/events/proteron/blockade.lua:76]: I'm guessing you wanted an `if` statement here.

**Comment by bobbens** [comment]
I've fixed it up and polished a bit and merged it. Thanks!

File: dat/events/proteron/blockade.lua (hunk 2 above)
Comment (Hraelzyr) [reply dat/events/proteron/blockade.lua:76]: I was trying to use the short-circuit fancy stuff in lua

**Pull request merged**